docs/aks-installation.md

```diff
@@ -0,0 +1,102 @@
+# Deploying Antrea on a AKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an Azure AKS cluster.
+
+## AKS Prerequisites
+
+Install the Azure Cloud CLI. Refer to [Azure CLI installation guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+## Creating the cluster
+
+You can use any method to create a AKS cluster. The example given here is using the Azure Cloud CLI.
+
+1. Create a AKS Cluster
+
+    ```bash
+    export RESOURCE_GROUP_NAME=aks-antrea-cluster
+    export CLUSTER_NAME=aks-antrea-cluster
+    export LOCATION=westus
+
+    az group create --name $RESOURCE_GROUP_NAME --location $LOCATION
+    az aks create \
+        --resource-group $RESOURCE_GROUP_NAME \
+        --name $CLUSTER_NAME \
+        --node-count 2 \
+        --network-plugin azure
+    ```
+
+    **Note** Do not specify network-policy option.
+
+2. Get AKS Cluster Credentials
+
+   ```bash
+   az aks get-credentials --name $CLUSTER_NAME --resource-group $RESOURCE_GROUP_NAME
+   ```
+
+3. Access your Cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                STATUS   ROLES   AGE     VERSION
+    aks-nodepool1-84330359-vmss000000   Ready    agent   6m21s   v1.16.10
+    aks-nodepool1-84330359-vmss000001   Ready    agent   6m25s   v1.16.10
+    ```
+
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy ``antrea-node-init`` DaemonSet to enable ``azure cni`` to operate in transparent mode.
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/vmware-tanzu/antrea/master/build/yamls/antrea-aks-node-init.yml
+    ```
+
+2. Download and Update Antrea YAML
```

Comment:
Does use need to update any thing in the yaml? If so please describe that.

Reply:
No manual config is required

Reply:
Then remove "Update", and maybe consider merge 2 and 3 - using a single command to get and apply the yaml. Check getting-started.md.